go/vt/vtgate/planbuilder/testdata/aggr_cases.json

```diff
@@ -3429,32 +3429,43 @@
       "Instructions": {
         "OperatorType": "Aggregate",
         "Variant": "Ordered",
-        "Aggregates": "sum_count(0) AS count(u.textcol1), sum_count(1) AS count(ue.foo)",
+        "Aggregates": "count(0) AS count(u.textcol1), count(1) AS count(ue.foo)",
```

Comment:
earlier the count was pushed down to the route, now all the rows are fetched at vtgate. should we optimize this?

Reply:
yeah, this is not an acceptable plan. we are fetching a LOT of data when we don't really need to